src/onnxruntime_utils.h

```diff
@@ -158,4 +159,34 @@ TRITONSERVER_Error* CompareDimsSupported(
     const std::vector<int64_t>& model_shape, const std::vector<int64_t>& dims,
     const int max_batch_size, const bool compare_exact);
 
+/// Parse the string as unsigned long integer.
+///
+/// \param value The string.
+/// \param parse_value The unsigned long integral value of the string.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_Error* ParseUnsignedLongValue(
```

Comment:
Can this be ParseUnsignedIntValue and you can add it to backend_common.cc along with the other parse utility functions.